criu/sk-unix.c

```diff
@@ -497,9 +497,29 @@ static int dump_one_unix_fd(int lfd, uint32_t id, const struct fd_parms *p)
 			goto err;
 		}
 
+		if (sk->wqlen != 0) {
```

Comment:
could you write comments for this and the next case that explain what exactly we don't support right now